code/game/gamemodes/objective.dm

```diff
@@ -403,6 +403,35 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 				counter++
 	return counter >= 8
 
+//wizard objectives (fairly hard to pull off and stupidly destructive to the station)
+
+/datum/objective/megafauna
+	name = "megafauna"
+	explanation_text = "Ensure a megafauna from the mining area is alive on or near the station at the end of the shift."
+	martyr_compatible = FALSE //while doable will make the objective way easier and instantly end round to finish martyr
+
+/datum/objective/megafauna/check_completion() //get a megafauna from the mining shuttle to the station. good luck!
```

Comment:
haven't we put in a bunch of countermeasures to keep anyone from doing this easily?

pretty much the only way to do this that I can think of is to ride a scooter as a sand golem and kite the blood drunk miner (since any of the actually threatening megafauna will straight up ice you) to the shuttle, or to abuse the invinicibility of weeping angels

Reply:
I'm not worried people do it without spells, remember that